hbase2/src/main/java/site/ycsb/db/hbase2/HBaseClient2.java

```diff
@@ -523,6 +557,15 @@ public Status delete(String table, String key) {
   void setConfiguration(final Configuration newConfig) {
     this.config = newConfig;
   }
+
+  private boolean isBooleanParamSet(String param, boolean defaultValue){
+    return Boolean.parseBoolean(getProperties().getProperty(param, Boolean.toString(defaultValue)));
+  }
+
+  private CompareOperator getCompareOperator(String operator) {
+    return CompareOperator.valueOf(operator.toUpperCase());
+  }
```

Comment:
this is only called in one place. replace the call with this implementation directly.